third_party/nvfuser/test/test_gpu_loop_rotation.cpp

```diff
@@ -0,0 +1,742 @@
+#include <gtest/gtest.h>
+
+#include <inlining.h>
+#include <ops/arith.h>
+#include <test/test_gpu_validator.h>
+#include <test/test_utils.h>
+
+namespace nvfuser {
+
+TEST_F(NVFuserTest, FusionLoopRotation1Inner_CUDA) {
+  // Please see note [Limitation of boundary assert]
+  EnableOutOfBoundAssert guard;
+
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto tv0 = makeConcreteTensor({-1, 3});
+  fusion.addInput(tv0);
+  auto tv1 = set(tv0);
+  auto tv2 = set(tv1);
+  auto tv3 = set(tv2);
+  auto tv4 = set(tv3);
+  fusion.addOutput(tv4);
+
+  inlineMost();
+  fusion.rotateLoop(tv4, -1, {tv1, tv2});
```

Comment:
This looks inconsistent to me. Both are a fusion-wide operation, and one implicitly uses the fusion from a `FusionGuard`, but another is a method of `Fusion`. I'd say just `rotateLoop(tv4, -1, {tv1, tv2});` would align well with the existing APIs.